experimental/ni_coq/README.md

```diff
@@ -0,0 +1,5 @@
+Experimental formalization of Oak + NI result.
+
+Instructions to re-generate makefile:
+(see also: https://coq.inria.fr/refman/practical-tools/utilities.html)
+coq_makefile -f _CoqProject -o Makefile
```

Comment:
I added a paragraph about noninterference and pointed to the two main files, but I'll iterate on it again to talk more about the progression as you requested.

I might like to hold off on talking about the file structure too much other than maybe the two main files for the runtime and the security definition at the moment because I bet I'll restructure it a lot and forget to update the readme.